src/cascade/model/grids.py

```diff
@@ -0,0 +1,235 @@
+"""Support for dismod concepts that are expressed over grids: age-time, smoothings and value priors.
+"""
+from math import isclose, isinf
+
+import numpy as np
+
+GRID_SNAP_DISTANCE = 1 / 365
```

Comment:
snap seems too large. Maybe 1/100 of this? b/c days matter in first age group, and 36500 inverted is still way above floating point differentiation.

Reply:
I'm going to push it down to epsilon.